exla/lib/exla/defn.ex

```diff
@@ -1449,6 +1473,31 @@ defmodule EXLA.Defn do
     {EXLA.Builder.build(res), update_outfeed(cache, comp_cache)}
   end
 
+  defp optional_computation(name, arg, expr, state, cache) do
```

Comment:
This seems to be the same as `while_computation`. Perhaps we can call it "maybe_token_computation" and reuse it on both?